terra/task.py

```diff
@@ -109,6 +100,9 @@ def __call__(self, *args, **kwargs):
         settings._wrapped.clear()
         settings._wrapped.update(self.translate_paths(self.request.settings,
             reverse_compute_volume_map, executor_volume_map))
+        # This is needed here, because I just loaded settings from a runner!
```

Comment:
Needed after all huh. The hardest line in logging.

Also, no comma before because